### ISSUE TITLE
Base Practitioner :: outstanding features

### ISSUE THREAD

**Issue by dtr-agency** [AuDigitalHealth/ci-fhir-r4#71]
# Prerequisites
- [X] I have searched [open](https://github.com/ci-fhir-r4/Issues/issues) and [closed](https://github.com/ci-fhir-r4/Issues/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed) issues to make sure it isn't already requested or reported
- [X] I have written a descriptive issue title

# The issue
## Change description
There are a number of outstanding features for Base Practitioner profile to be implemented:

1.  canonical URL is non-FHIR version specific
2. StructureDefinition.description and Practitioner.short improved in line with current language
3. Practitioner.active matches the decision log Constraints on element active in Agency CI FHIR profiles

In the implementation guides the introductory material is largely placeholder; this needs to be resolved in order to provide clarity on the purpose, use, and guidance on implementation for this profile.

Additionally the known issues should be brought into line with the clarity and completeness of other profiles.

## What it actually enables people to do
Basically have a working profile.

## How awesome would it be?
Very - the features are targeted towards keeping this profile functioning.
The informative improvements provide a better basis for stakeholder engagement, future work and clarity in demonstrating how the overarching usage scenarios in the implementation guide would be served.

## Workarounds
N/A

## Additional context
- Recent changes in the style and content to be added in  informative material is demonstrated in recent work for Patient with Mandatory Identifier as part of https://github.com/AuDigitalHealth/ci-fhir-r4/issues/4.
- [Use FHIR version independent canonical URLs for Agency profiles #17](https://github.com/AuDigitalHealth/ci-fhir-r4/issues/17)

**Comment by dbojicic-agency** [comment]
Work on addressing these issues and improving introductory material for this profile has started via internal ticket CIFMM-3485.

**Comment by dbojicic-agency** [comment]
Changes made so far:
- canonical URL change (removing 4.0) to non-FHIR version specific http://ns.electronichealth.net.au/ci/fhir/StructureDefinition/practitioner-dh-base-1
- combined inv-dh-prac-02 and inv-dh-prac-03 into one, and (inv-dh-prac-02) with updated invariant description and FHIRPath expression
- removed must support flag from Practitioner.active and and fixed value to 'true'
- improved StructureDefinition.description and StructureDefinition.differential.short
- added first draft of introductory material and implementation guidance
- refined list of known issues. 
 
Walked through with the product owner and agreed on further changes to:
- further improve introductory content
- remove must support flag from Practitioner.birthDate element 
- keep must support flag on Practitioner.gender element as requiered towards supporting Cultural and Linguistic Diversity (CALD) in electronic health information exchange, see GitHub issue [#13](https://github.com/AuDigitalHealth/ci-fhir-r4/issues/13) data set in electronic health information exchange 
- add known issue about terminology for Practitioner.qualification

**Comment by dbojicic-agency** [comment]
Ready for another walk through with product owner after introducing boundaries and relationships introductory content and new qualification terminology known issue, and removing must support flag from birthDate.

**Comment by dbojicic-agency** [comment]
Product owner accepted StructureDefinition and introductory content in the Diagnostic Report FHIR IG and would like a qualification related known issue updated before the story can be sent for peer review. 
Practitioner qualification terminology in Base Practitioner issue in GitHub [GitHub #90](https://github.com/AuDigitalHealth/ci-fhir-r4/issues/90) also needs to be updated to include more details about work done so far and the future direction.

**Comment by dbojicic-agency** [comment]
Sent for peer review to @RichardTON via internal story [CIFMM-3485](https://jira.digitalhealth.gov.au/browse/CIFMM-3485).

**Comment by dbojicic-agency** [comment]
Pull request has been approved for merge into master.
